modules/php/Cards/Rules/RuleInflation.php

```diff
@@ -16,13 +16,28 @@ public function __construct($cardId, $uniqueId)
     );
   }
 
-  public function immediateEffectOnPlay($player)
+  public function immediateEffectOnPlay($player_id)
   {
-    // @TODO : set game state?
+    Utils::getGame()->setGameStateValue("activeInflation", 1);
+    // Draw rule is adapted immediately, so current player draws an extra card
+    Utils::getGame()->performDrawCards($player_id, 1);
```

Comment:
If poor bonus or party bonus apply when playing the inflation. Should we not actually draw 2 (or 3) cards here?

Reply:
I'm asking the question via email. If I get an answer today, I'll do the change if need be and will merge.

Reply:
I am pretty sure you are right, inflation should be applied to the Draw rule and to any bonuses also.
This is also the way I do it when they are already on the table.